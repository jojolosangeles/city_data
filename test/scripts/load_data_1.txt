sample20 = data/sample_20_Traffic_Collision_Data_from_2010_to_Present.csv
sample20.columns
