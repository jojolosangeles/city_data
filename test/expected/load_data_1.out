Test: ../pd.py
> load data/sample_20_Traffic_Collision_Data_from_2010_to_Present.csv as sample20
> columns sample20
['DR Number' 'Date Reported' 'Date Occurred' 'Time Occurred' 'Area ID'
 'Area Name' 'Reporting District' 'Crime Code' 'Crime Code Description'
 'MO Codes' 'Victim Age' 'Victim Sex' 'Victim Descent' 'Premise Code'
 'Premise Description' 'Address' 'Cross Street' 'Location']
Test completed: ../pd.py
